libbpf-tools/runqlen.c

```diff
@@ -45,17 +45,19 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
 const char argp_program_doc[] =
 "Summarize scheduler run queue length as a histogram.\n"
 "\n"
-"USAGE: runqlen [--help] [-C] [-O] [-T] [interval] [count]\n"
+"USAGE: runqlen [--help] [-C] [-O] [-T] [-f FREQUENCY] [interval] [count]\n"
 "\n"
 "EXAMPLES:\n"
 "    runqlen         # summarize run queue length as a histogram\n"
 "    runqlen 1 10    # print 1 second summaries, 10 times\n"
 "    runqlen -T 1    # 1s summaries and timestamps\n"
 "    runqlen -O      # report run queue occupancy\n"
-"    runqlen -C      # show each CPU separately\n";
+"    runqlen -C      # show each CPU separately\n"
+"    runqlen -f 199  # sample as 199hz\n";
```

Comment:
typo: sample at 199HZ

Reply:
Fixed, thanks! Also fixed this in https://github.com/iovisor/bcc/pull/3131.